templates/navigation/navbar.html

```diff
@@ -0,0 +1,53 @@
+<div class="navbar js-nav-expandable">
+  <div class="navbar__container">
+    <div class="navbar__inner">
+      <button class="navbar__button js-nav-mobile-closable">
+        <span class="navbar__text">menu</span>
+        <svg class="i-svg i-svg--menu"><use xlink:href="#i-menu" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg>
+        <svg class="i-svg i-svg--close"><use xlink:href="#i-close" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg>
+      </button>
+
+      <div class="navbar__group js-nav-group-viewable js-nav-item-expandable">
+        <nav id="navigation" role="navigation" class="navigation">
+          <ul class="basic-main-menu menu">
+            <li class="menu-item menu-item--collapsed">
```

Comment:
I'm not sure `menu-item-collapsed` is what you want here. I am having troubles getting the drop downs to work and I think this may be why. It may need to change to `menu-item--expanded`.